src/Middleware/ErrorHandler.php

```diff
@@ -43,6 +45,21 @@ public function process(
             $response = $handler->handle($request);
         } catch (HttpException $e) {
             $response = $this->createResponse($e->getMessage(), $e->getStatusCode(), $e->getHeaders());
+        } catch (ValidationException $e) {
+            $response = $this->createResponse('', 302, ['Location' => $this->getPreviousUrl($request)]);
+
+            if ($request instanceof Request) {
+                $session = $request->getSession();
+                $session->set(
+                    'errors',
+                    array_merge_recursive(
+                        $session->get('errors', []),
+                        ['validation' => $e->getValidator()->getErrors()]
+                    )
+                );
+
+                $session->set('form-data', $request->request->all());
```

Comment:
Das schreibt beim Login Passwörter in die Session! Wird das benötigt? Kann man Passwörter aus dem Request filtern?

Reply:
Könnte man tun, welche Felder sollten alle sonst noch raus genommen werden?

Reply:
Passwörter, API-Token, alles was Rechte verschafft. Eventuell auch Dateien aus Uploads. Alles andere sollte passen.